src/emitter.cpp

```diff
@@ -2,28 +2,33 @@
 Copyright (c) 2022 Snowplow Analytics Ltd. All rights reserved.
 
 This program is licensed to you under the Apache License Version 2.0,
-and you may not use this file except in compliance with the Apache License Version 2.0.
-You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+and you may not use this file except in compliance with the Apache License
+Version 2.0. You may obtain a copy of the Apache License Version 2.0 at
+http://www.apache.org/licenses/LICENSE-2.0.
 
 Unless required by applicable law or agreed to in writing,
 software distributed under the Apache License Version 2.0 is distributed on an
-"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+implied. See the Apache License Version 2.0 for the specific language governing
+permissions and limitations there under.
 */
 
 #include "emitter.hpp"
 
+using namespace snowplow;
+using std::invalid_argument;
 using std::lock_guard;
 using std::stringstream;
-using std::invalid_argument;
 using std::unique_lock;
 
-const int post_wrapper_bytes = 88; // "schema":"iglu:com.snowplowanalytics.snowplow/payload_data/jsonschema/1-0-4","data":[]
-const int post_stm_bytes = 22;     // "stm":"1443452851000"
-
-Emitter::Emitter(const string & uri, Method method, Protocol protocol, int send_limit, 
-  int byte_limit_post, int byte_limit_get, const string & db_name) : m_url(this->get_collector_url(uri, protocol, method)) {
+const int post_wrapper_bytes =
+    88;  // "schema":"iglu:com.snowplowanalytics.snowplow/payload_data/jsonschema/1-0-4","data":[]
```

Comment:
Another point with I'm twisted by the rule.
In this case I wouldn't wrap with with the value in the newline. Also the comment would exceed in any case in the new line.

My preference would be for something like this:
```suggestion
// bytes length for schema
// (e.g.: "schema":"iglu:com.snowplowanalytics.snowplow/payload_data/jsonschema/1-0-4","data":[])
const int post_wrapper_bytes = 88;  
```
Even if I still prefer the original version.